client/src/pages/components/categories/CategoryScreen.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
+import AppLayout from "../../../Layouts/AppLayout";
+import {
+  Button,
+  Paper,
+  Box,
+  Modal,
+  Typography,
+  TextField,
+  Grid,
+} from "@mui/material";
+import AddBoxIcon from "@mui/icons-material/AddBox";
+import ClassIcon from "@mui/icons-material/Class";
+import CategoryTable from "./CategoryTable";
+import apiClient from "../../../utils/axios";
+import useSWR, { mutate } from "swr";
+import CategoryApi from "./index";
+
+import { useSnackbar } from "notistack";
+import { useForm, Controller } from "react-hook-form";
+
+const CategoryScreen = ({ children }) => {
+  const [name, setName] = useState("");
+  const [description, setDesription] = useState("");
+
+  const { enqueueSnackbar } = useSnackbar();
+
+  const [open, setOpen] = useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  const style = {
```

Comment:
this style is redundant in some pages. please make a common css folder to separate this.